nirodents/workflows/brainextraction.py

```diff
@@ -211,8 +211,8 @@ def init_rodent_brain_extraction_wf(
         ])
 
     # Spatial normalization step
-    lap_tmpl = pe.Node(ImageMath(operation="Laplacian", op2="0.4 1.0"), name="lap_tmpl")
-    lap_target = pe.Node(ImageMath(operation="Laplacian", op2="0.4 1.0"), name="lap_target")
+    lap_tmpl = pe.Node(ImageMath(operation="Laplacian", op2="0.4 1"), name="lap_tmpl")
+    lap_target = pe.Node(ImageMath(operation="Laplacian", op2="0.4 1"), name="lap_target")
```

Comment:
Sorry, I wasn't clear. When I had the integer 1, rather than 1.0, the value normalisation _didn't_ work. If you checked the Laplacian and it was between 0 and 1 with the float, we should stick with the float.

Reply:
oh, it doesn't seem to normalize anything either way

Reply:
let's address this separately